installer/resources/pacbot_app/files/DB.sql

```diff
@@ -1225,9 +1225,9 @@ INSERT IGNORE INTO `cf_Target` (`targetName`,`displayName`,`targetDesc`,`categor
 INSERT IGNORE INTO `cf_Target` (`targetName`,`displayName`,`targetDesc`,`category`,`dataSourceName`,`targetConfig`,`status`,`userId`,`endpoint`,`createdDate`,`modifiedDate`,`domain`) VALUES ('pubsub','Pub/Sub','GCP pub sub topic','Compute','gcp','{"key":"id","id":"id"}','enabled','admin',concat(@eshost,':',@esport,'/gcp_pubsub/pubsub'),'2022-06-29','2022-06-29','Infra & Platforms');
 INSERT IGNORE INTO `cf_Target` (`targetName`,`displayName`,`targetDesc`,`category`,`dataSourceName`,`targetConfig`,`status`,`userId`,`endpoint`,`createdDate`,`modifiedDate`,`domain`) VALUES ('dataproc','Dataproc Cluster','GCP dataproc clusters','Compute','gcp','{"key":"id","id":"id"}','enabled','admin',concat(@eshost,':',@esport,'/gcp_dataproc/dataproc'),'2022-07-19','2022-07-19','Infra & Platforms');
 INSERT IGNORE INTO `cf_Target` (`targetName`,`displayName`,`targetDesc`,`category`,`dataSourceName`,`targetConfig`,`status`,`userId`,`endpoint`,`createdDate`,`modifiedDate`,`domain`) VALUES ('clouddns','Cloud DNS','GCP Cloud DNS','Google Cloud','gcp','{"key":"id","id":"id"}','enabled','admin',concat(@eshost,':',@esport,'/gcp_clouddns/clouddns'),'2022-06-01','2022-06-01','Infra & Platforms');
-INSERT IGNORE INTO `cf_Target` (`targetName`,`displayName`,`targetDesc`,`category`,`dataSourceName`,`targetConfig`,`status`,`userId`,`endpoint`,`createdDate`,`modifiedDate`,`domain`) VALUES ('cloudsql_sqlserver','SQL Server','GCP Cloud SQL Server','CloudSQLServer','gcp','{"key":"id","id":"id"}','enabled','admin',concat(@eshost,':',@esport,'/gcp_cloudsql_sqlserver'),'2022-06-29','2022-06-29','Infra & Platforms');
-INSERT IGNORE INTO `cf_Target` (`targetName`,`displayName`,`targetDesc`,`category`,`dataSourceName`,`targetConfig`,`status`,`userId`,`endpoint`,`createdDate`,`modifiedDate`,`domain`) VALUES ('cloudsql_mysqlserver','MySQL Server','GCP Cloud MySQL Server','CloudMySQLServer','gcp','{"key":"id","id":"id"}','enabled','admin',concat(@eshost,':',@esport,'/gcp_cloudsql_mysqlserver'),'2022-10-04','2022-10-04','Infra & Platforms');
-INSERT IGNORE INTO `cf_Target` (`targetName`,`displayName`,`targetDesc`,`category`,`dataSourceName`,`targetConfig`,`status`,`userId`,`endpoint`,`createdDate`,`modifiedDate`,`domain`) VALUES ('cloudsql_postgres','PostgreSQL','GCP Cloud postgres Server','CloudPostgresServer','gcp','{"key":"id","id":"id"}','enabled','admin',concat(@eshost,':',@esport,'/gcp_cloudsql_postgres'),'2022-10-07','2022-10-07','Infra & Platforms');
+INSERT IGNORE INTO `cf_Target` (`targetName`,`displayName`,`targetDesc`,`category`,`dataSourceName`,`targetConfig`,`status`,`userId`,`endpoint`,`createdDate`,`modifiedDate`,`domain`) VALUES ('cloudsql_sqlserver','SQL Server','GCP Cloud SQL Server','CloudSQLServer','gcp','{"key":"id,databaseVersion","id":"id"}','enabled','admin',concat(@eshost,':',@esport,'/gcp_cloudsql_sqlserver'),'2022-06-29','2022-06-29','Infra & Platforms');
```

Comment:
How can create date be in the past, please correct

Reply:
Done.